ownCloud/Viewer/DisplayViewProtocol.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  DisplayViewProtocol.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 29/08/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+import UIKit
+import ownCloudSDK
+
+protocol DisplayViewProtocol where Self: DisplayViewController {
+
+	static var features: [String : Any]? {get}
+	static var supportedMimeTypes: [String] {get}
+}
+
+struct FeatureKeys {
+	static let canEdit: String = "featureKeyCanEdit"
+	static let canSave: String = "featureKeyCanSave"
+	static let showPDF: String = "featureKeyShowPDF"
```

Comment:
As far as I can see, we don't need `showPDF` as a feature. Instead, PDF viewer(s) can be found by querying `OCExtensionManager` with the PDF MIME-Type `application/pdf`.